lib/quests/json/piggy_nibbler.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "Q11",
+  "title": "Meat Galore",
+  "description": "Go nibble 6 pigs",
+  "requirements": [
+    {
+      "id": "Q11R2",
```

Comment:
This will work fine, but we could also just change it to Q11R1. The only requirement here is that the id is unique among all quests. So usually I just created a string from the quest id and the requirement number. Since this is the first requirement in this quest, I would tend to call it R1.